platform-emf/src/main/java/com/softicar/platform/emf/token/parser/converter/EmfTokenToByteArrayConverter.java

```diff
@@ -0,0 +1,22 @@
+package com.softicar.platform.emf.token.parser.converter;
+
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.db.runtime.field.DbByteArrayField;
+import com.softicar.platform.emf.EmfI18n;
+import java.util.Base64;
+
+class EmfTokenToByteArrayConverter extends AbstractEmfTokenConverter<byte[], DbByteArrayField<?>> {
```

Comment:
Note to myself: This needs to consider maxlength and lengthbits (much like the String converter).